### DISCUSSION TITLE
Ask For Support

### DISCUSSION THREAD

**Discussion by SaptarshiSarkar12** [SaptarshiSarkar12/Drifty#241] [Q&A] [answered]
**Here, anyone can ask for help regarding the [project](https://github.com/SaptarshiSarkar12/Drifty) to the community.
Please ask your questions with adequate details, and adhere to the [Code Of Conduct](https://github.com/SaptarshiSarkar12/Drifty/blob/master/CODE_OF_CONDUCT.md).**
Read the project [Contributing Guidelines](https://github.com/SaptarshiSarkar12/Drifty/blob/master/CONTRIBUTING.md).

**Comment by Tushar1323** [comment]
Hello there👋 I forked and cloned the Drifty repository in my local disk, but after that I am stuck that how to run this website in my local environment.
Can your share me a correct way for running the drift in local environment?

**Reply by SaptarshiSarkar12** [reply, marked as answer]
Okay @Tushar1323. Follow these steps -
- Go to `Website` directory
- Run `npm ci` to install the dependencies
- Run `npm run dev` to start the development server
- Open [`localhost:3000`](http://localhost:3000/) in your browser to see the live website
